### PR TITLE
feat: replace styled-components with emotion

### PR DESCRIPTION
**Pull request by amartya-ws**
- replace styled-components with emotion pkg

### CODE REVIEW

**Review by christin-wednesday — COMMENTED** [review]
Why are we moving to emotion ?

snapshot serializers look nice to have  https://emotion.sh/docs/@emotion/jest

**Comment by amartya-ws** [comment]
> Why are we moving to emotion ?
> 
> snapshot serializers look nice to have https://emotion.sh/docs/@emotion/jest

emotion is lighter than styled-components I guess, helps in reducing the build size

**Comment by praveenkumar1798** [comment]
CI is failing

**Comment by amartya-ws** [comment]
> CI is failing

fixed

**Review by praveenkumar1798 — APPROVED** [review]

File: app/themes/tests/fonts.test.js [hunk 1]
```diff
@@ -2,29 +2,82 @@ import fonts from '../fonts';
...
+        .big()
+        .styles.replace(/[\r\n\s]+/gm, '')
+        .trim()
+    ).toBe('font-size:1.75rem;');
+    expect(
+      fonts.size
+        .large()
+        .styles.replace(/[\r\n\s]+/gm, '')
+        .trim()
+    ).toBe('font-size:2.25rem;');
```
Comment (praveenkumar1798) [inline app/themes/tests/fonts.test.js:28]: expect(fonts.size.large()['styles']).toEqual('font-size:2.25rem');

**Pull request merged**